examples/hosted-delivery-function-potterdb/contentful-app-manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "deliveryFunctions": [
+    {
+      "id": "PotterDB",
+      "name": "Example Delivery Function querying Potter-DB GraphQL API",
```

Comment:
nit: You could move this to description and just call it "PotterDB Delivery Function"